opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -18,6 +18,24 @@
     Meter,
     MeterProvider,
 )
+from opentelemetry.sdk._metrics._internal.aggregation import (  # noqa: F401
+    Aggregation,
+    AggregationTemporality,
+    DefaultAggregation,
+    DropAggregation,
+    ExplicitBucketHistogramAggregation,
+    LastValueAggregation,
+    SumAggregation,
+)
+from opentelemetry.sdk._metrics._internal.instrument import (  # noqa: F401
+    Counter,
+    Histogram,
+    ObservableCounter,
+    ObservableGauge,
+    ObservableUpDownCounter,
+    UpDownCounter,
+)
+from opentelemetry.sdk._metrics._internal.view import View  # noqa: F401
```

Comment:
I'd be OK to have View and Aggregation stuff L21-28 in a `opentelemetry.sdk._metrics.view` module if you wanted to separate it out more. Up to you

Reply:
Added `view` module.